assets/js/composer.js

```diff
@@ -292,13 +294,66 @@ PageComposer.prototype = {
         formGroup.appendChild(errorWrapper);
       }
 
-      const errorItem = document.createElement('li');
-      errorItem.innerHTML = `<i class="fas fa-exclamation-circle" aria-hidden="true"></i> ${violation.title}`;
+      const errorItem = this.createErrorItem(violation.title);
 
       errorList.appendChild(errorItem);
     });
   },
 
+  /**
+   * In case the field is a collection,
+   * we need to go directly to the `.form-group` in order to display the error.
+   *
+   * @param violation
+   */
+  handleBlockCollectionErrors(violation) {
```

Comment:
@Daric971 

Did you copy this function from somewhere,  or did you do it from scratch?